pyaldata/utils.py

```diff
@@ -543,3 +543,47 @@ def extract_interval_from_signal(trial_data, signal, epoch_fun):
     list of the extracted np.arrays
     """
     return [trial[signal][epoch_fun(trial), ...] for (i, trial) in trial_data.iterrows()]
+
+
+def remove_cmp_formatting(s):
+    """
+    Used in read_cmp() to remove formatting in .cmp file
+    
+    Parameters
+    ----------
+    s: str
+        one line in the file
+        
+    Returns
+    -------
+    list of strings
+    """
+    for r in (('\t', ' '), ('\n', ''), ('elec', '')):
+        s = s.replace(*r)       
+    return s.split() 
+
+
+def read_cmp(file_path):
+    """
+    Read in Blackrock Microsystems .cmp file into Python
+    
+    Parameters
+    ----------
+    file_path: str
+        .cmp file path + name 
+        
+    Returns
+    -------
+    df_array: dataframe of shape (num electrodes, 5)
+        [col (int), row (int), channel number (str), within_channel_num (int), global electrode number (int)]
+    """
+    # Open file, remove comments and remove other formatting we don't need
+    f = open(file_path)
+    temp = [line for line in f.readlines() if not line.startswith('//')] 
+    clean_lsts = [remove_cmp_formatting(l) for i, l in enumerate(temp) if not i == 0] 
```

Comment:
```suggestion
    clean_lsts = [remove_cmp_formatting(l) for l in temp[1:]]
```
You just want to skip the first line, right?

Reply:
correct